codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -142,8 +162,8 @@ class FileCodesignVisitor {
     required FileSystemEntity zipEntity,
     required String entitlementParentPath,
   }) async {
-    log.info('This embedded file is ${zipEntity.path} and entilementParentPath is $entitlementParentPath\n');
-    final String currentFileName = zipEntity.path.split('/').last;
+    log.info('This embedded file is ${zipEntity.path} and entilementParentPath is $entitlementParentPath');
```

Comment:
```suggestion
    log.info('This embedded file is ${zipEntity.path} and entitlementParentPath is $entitlementParentPath');
```

Reply:
My bad, I did a replace all on entilement (a messed up word). Thanks for catching it!